app/views/log_entries/index.haml

```diff
@@ -22,6 +28,9 @@
                 = f.text_area :text, required: true, rows: 8, hide_label: true
                 = f.primary 'Change log entry'
```

Comment:
```suggestion
                 .d-flex
                    = f.primary 'Create log entry'
                    .input-group.ml-2
                      .input-group-text
                        .custom-control.custom-checkbox
                          = f.check_box_without_bootstrap :pinned, class: 'custom-control-input'
                          = f.label :pinned, class: 'custom-control-label'
```
We also need to be able to pin existing entries, so we might as well copy what we have to the edit form as well rather than just have a dedicated unpin button